Terasology/CombatSystem
# Switch to using StartingInventroySystem

Switch ```CombatSystem``` to use the ```StartingInventorySystem```. 

**How To Test**

Load a game using ```CombatSystem``` as the gameplay module and verify that the starting inventory is the same on both.
